### ISSUE TITLE
audacity not able to run

### ISSUE THREAD

**Issue by moh0009** [audacity/audacity#5735]
### Bug description

hi audacity developers i am trying to open audacity 3.4.2 but i got this :
``` 
:~$ '/home/user/Downloads/audacity-linux-3.4.2-x64.AppImage' 
/lib/x86_64-linux-gnu/libatk-1.0.so.0
/lib/x86_64-linux-gnu/libatk-bridge-2.0.so.0
/lib/x86_64-linux-gnu/libcairo-gobject.so.2
/lib/x86_64-linux-gnu/libcairo.so.2
/lib/x86_64-linux-gnu/libgio-2.0.so.0
/lib/x86_64-linux-gnu/libglib-2.0.so.0
/lib/x86_64-linux-gnu/libgmodule-2.0.so.0
/lib/x86_64-linux-gnu/libgobject-2.0.so.0
/lib/x86_64-linux-gnu/libgthread-2.0.so.0
/lib/x86_64-linux-gnu/libjack.so.0
/lib/x86_64-linux-gnu/libpixman-1.so.0
findlib: libportaudio.so: cannot open shared object file: No such file or directory
/home/user/Downloads/audacity-linux-3.4.2-x64.AppImage: Using fallback for library 'libportaudio.so'
/tmp/.mount_audaciOpoEmd/bin/audacity: /tmp/.mount_audaciOpoEmd/lib/libselinux.so.1: no version information available (required by /lib/x86_64-linux-gnu/libgio-2.0.so.0)
```

os : ubuntu 22.04

### Steps to reproduce

i open the app form terminal

### Expected behavior

to open the main interface 

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Linux

### Additional context

![image](https://github.com/audacity/audacity/assets/83363934/a2342209-ab32-4e07-bc0b-8e6bc5e5016c)

**Comment by moh0009** [comment]
fixed in latest version

**Comment by Spielmops** [comment]
hartmut@Jupiter~>./audacity-linux-3.5.1-x64.AppImage 
/usr/lib64/libatk-1.0.so.0
/usr/lib64/libatk-bridge-2.0.so.0
/usr/lib64/libcairo-gobject.so.2
/usr/lib64/libcairo.so.2
/usr/lib64/libgio-2.0.so.0
/usr/lib64/libglib-2.0.so.0
/usr/lib64/libgmodule-2.0.so.0
/usr/lib64/libgobject-2.0.so.0
/usr/lib64/libgthread-2.0.so.0
/usr/lib64/libjack.so.0
/usr/lib64/libpixman-1.so.0
findlib: libportaudio.so: cannot open shared object file: No such file or directory
/home/hartmut/audacity-linux-3.5.1-x64.AppImage: Using fallback for library 'libportaudio.so'
/tmp/.mount_audaciX7Vvav/bin/audacity: /tmp/.mount_audaciX7Vvav/lib/libselinux.so.1: no version information available (required by /usr/lib64/libgio-2.0.so.0)

Spielmops
